test/udev_auto_test.rb

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env rspec
+
+require_relative "test_helper"
+
+require "yast"
+
+include Yast
+
+Yast.import "LanUdevAuto"
+
+describe "LanUdevAuto#Write" do
```

Comment:
```
$ ls -1 test/*udev*test.rb
test/lan_udev_auto_test.rb
test/udev_auto_test.rb
test/udev_test.rb
```

Also tests should be well organized, so I think this file should be folded into lan_udev_auto_test.rb